### ISSUE TITLE
Add <div> and <span> wrappers around edits

### ISSUE THREAD

**Issue by frostyweather** [bradfrost/atomic-design#117]
Add `<div>` and `<span>` wrappers around the edits to style these chunks of text differently than the rest of the book. Just letting you know, @FullCreamMilkMan.

**Comment by FullCreamMilkMan** [comment]
Hi. Sure, OK. Do you have particular edits in mind, or just generally? Could you show me, so I know exactly what you mean? Thanks, @frostyweather.

**Comment by frostyweather** [comment]
Sure thing!  Basically all I am doing is removing the double brackets [[]] and replacing those with `<div class="editor">` for the remarks that are their own paragraph or `<span class="editor">` for the double brackets within a paragraph. I am not changing any of the edit material, just changing the styling of how they look on the page. For example:

I am changing this:
```
[[*Ed: Is "custom-tailored" like belt and braces? We can ditch "custom-" as "tailored" can do all the work.*]]
```

To this:
```
<div class="editor">
{% capture m %}
Ed: Is "custom-tailored" like belt and braces? We can ditch "custom-" as "tailored" can do all the work.
{% endcapture %}
{{ m | markdownify }}
</div>
```

I am doing this to basically give all of your edits a class name so I can turn the text red or give it a highlighted look to differentiate it from the actual content of the book. Sound good?

![screen shot 2016-04-13 at 10 02 07 am](https://cloud.githubusercontent.com/assets/13662843/14495823/0faa47ac-015f-11e6-8a2c-c63474f262b0.png)

**Comment by FullCreamMilkMan** [comment]
Yeah, that's fine. When Brad has finished chapter 5, do you want me to wrap my edits like this, or can I leave that to you?

**Comment by FullCreamMilkMan** [comment]
Oh, and there are a couple of edits which I deliberately marked up so they remain hidden. Have you identified them? They're early on, I think, and only once or twice. They should stay hidden, though I can discuss that with Brad.

**Comment by frostyweather** [comment]
@FullCreamMilkMan You can leave that to me if it is easier doing it the other way. I made the invisible ones visible I believe. Probably a good question to ask @bradfrost whether he wants these hidden or not.

**Comment by frostyweather** [comment]
@FullCreamMilkMan Also, I created a new branch for this called edits-ian so we can always compare and contrast with the edits branch in case we do want to go back to the invisible state.

**Comment by FullCreamMilkMan** [comment]
I'll keep using the square braces for now – it'll be much easier for me and won't slow me down while editing. Hope that's OK. Thanks for setting up a new branch for your work. That'll keep things nice and neat, and like you say we can revert back as and when necessary. Great. 

Now I can start to get nervous as my usually unseen work prepares for the light of day!

**Comment by frostyweather** [comment]
@FullCreamMilkMan Sounds good. That works for me. Your work will look great! haha

**Comment by bradfrost** [comment]
@FullCreamMilkMan @frostyweather 

> Oh, and there are a couple of edits which I deliberately marked up so they remain hidden. Have you identified them? They're early on, I think, and only once or twice. They should stay hidden, though I can discuss that with Brad.

If you feel like those should be hidden then we can do that. I'd ask why they're hidden (are you calling me ugly?) but if you feel strongly about keeping a few comments private, that's totally fine. 

> Now I can start to get nervous as my usually unseen work prepares for the light of day!

Hahaha it'll be fine!
